sadu-updater/src/main/java/de/chojo/sadu/updater/SqlUpdater.java

```diff
@@ -205,6 +206,9 @@ private void performUpdate(Patch patch) throws SQLException {
             for (var query : type.splitStatements(patch.query())) {
                 try (var statement = conn.prepareStatement(adjust(query))) {
                     statement.execute();
+                } catch (SQLException e) {
+                    log.warn("Failed to execute statement: \n %s".formatted(query), e);
```

Comment:
```suggestion
                    log.warn("Failed to execute statement:\n{}", query, e);
```
You can simplify this with log4j ^^

Reply:
Oh, didn't know that :)
It loops cleaner 👍